applications/asset_tracker_v2/tests/location_module/CMakeLists.txt

```diff
@@ -1,32 +1,31 @@
 #
-# Copyright (c) 2021 Nordic Semiconductor
+# Copyright (c) 2022 Nordic Semiconductor
```

Comment:
```suggestion
# Copyright (c) 2021 Nordic Semiconductor
```

Copyright year should stay first year of publication of the file.

Reply:
Github sees this as renaming although it's deleting gnss module tests and adding location module tests so I'd say this should be 2022.

Reply:
@coderbyheart Please check this and resolve if the response is fine or comment why it's not.